test/DMLTest.php

```diff
@@ -468,4 +468,13 @@ public function test_18_02_delete_loop() {
         gc_collect_cycles();
         gc_collect_cycles();
     }
+
+	/**
+	 * @doesNotPerformAssertions
+	 */
+	public function test_19_schema_obtain_failed() {
+        $port = TestHelpers::getTarantoolPort();
+		$empty_tarantool = new Tarantool('localhost', $port, 'test');
```

Comment:
`test` password is forgotten.

Reply:
It seems, if only username is passed (but no password) the client will silently connect as 'guest' user without a password. What a hell…

If I change the following `&&` to `||`:

https://github.com/tarantool/tarantool-php/blob/e67cb231d48219d1f01cf1e5b4c3cc93d8004f9f/src/tarantool.c#L309

Then I got `TarantoolClientError: Invalid MsgPack - authentication request body`.

That's all looks as a bug.

Reply:
Fixed. I'll file a separate issue re behaviour when a password is not provided.

Reply:
Filed #165.